doc/planning_scene_python/src/planning_scene_tutorial.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
+import sys
+
+def main():
+    rospy.init_node("planning_scene_tutorial")
+    moveit_commander.roscpp_initialize(sys.argv)
+
+    # BEGIN_TUTORIAL
+    #
+    # Setup
+    # ^^^^^
+    #
+    # The :moveit_core_python:`core.planning_scene.PlanningScene` class can be easily setup and
+    # configured using a URDF and
+    # SRDF. This is, however, not the recommended way to instantiate a
+    # PlanningScene. At the time of writing there are not yet python bindings
+    # for the PlanningSceneMonitor, which is is the recommended method to
+    # create and maintain the current planning scene
+    # using data from the robot's joints and the sensors on the robot. In
+    # this tutorial, we will instantiate a PlanningScene class directly,
+    # but this method of instantiation is only intended for illustration.
+    import moveit_commander
+    import numpy as np
+    import rospy
+    from geometry_msgs.msg import PoseStamped
+    import moveit.core
+    from moveit.core import kinematic_constraints, collision_detection, robot_model
+    from moveit.core.planning_scene import PlanningScene
+
+    r = rospkg.RosPack()
+    urdf_path = pathlib.Path(r.get_path("moveit_resources_panda_description")) / "urdf" / "panda.urdf"
+    srdf_path = pathlib.Path(r.get_path("moveit_resources_panda_moveit_config")) /"config"/"panda.srdf"
+    kinematic_model = moveit.core.load_robot_model(urdf_path.as_posix(), srdf_path.as_posix())
+    planning_scene = PlanningScene(kinematic_model, collision_detection.World())
+    current_state = planning_scene.getCurrentStateNonConst()
+    joint_model_group = current_state.getJointModelGroup("panda_arm")
+
+    # Collision Checking
+    # ^^^^^^^^^^^^^^^^^^
+    #
+    # Self-collision checking
+    # ~~~~~~~~~~~~~~~~~~~~~~~
+    #
+    # The first thing we will do is check whether the robot in its
+    # current state is in *self-collision*, i.e. whether the current
+    # configuration of the robot would result in the robot's parts
+    # hitting each other. To do this, we will construct a
+    # :moveit_core_python:`core.collision_detection.CollisionRequest` object and a
+    # :moveit_core_python:`core.collision_detection.CollisionResult` object and pass them
+    # into the collision checking function. Note that the result of
+    # whether the robot is in self-collision or not is contained within
+    # the result. Self collision checking uses an *unpadded* version of
```

Comment:
This sentence is confusing to me. Does this capture what you intend?
```suggestion
    # into the collision checking function. Note that 
    # whether the robot is in self-collision or not is contained within
    # the `CollisionResult` object. Self collision checking uses an *unpadded* version of
```